common/src/main/java/org/arpha/dto/product/CategoryDto.java

```diff
@@ -0,0 +1,17 @@
+package org.arpha.dto.product;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class CategoryDto {
```

Comment:
It's not actually clear what exactly should be done with this dto. Is it create/update request?